cycode/cli/printers/sca_table_printer.py

```diff
@@ -112,7 +112,7 @@ def _print_summary_issues(detections: List, title: str) -> None:
     def _shortcut_dependency_paths(dependency_paths: str) -> str:
         dependencies = dependency_paths.split(' -> ')
 
-        if len(dependencies) < 2:
+        if len(dependencies) < 3:
```

Comment:
what is the magic value?

Reply:
The dependency path should be mention only direct dependency until the problematic dependency.
If we have lower than 3, no changes should be on dependency paths, otherwise we need to remove all dependencies between direct dependencies to problematic dependency

Reply:
why do you need to remove path to problematic dependency?

Reply:
all the middle dependencies between the direct dependency to vulnerable dependency since this path could be very long

Reply:
it's a table output, right?
why not move dependency to a new line?
e.g. `A` or `B` option instead of `C`
<img width="182" alt="image" src="https://github.com/cycodehq-public/cycode-cli/assets/25585934/09cecfef-abbc-4460-ba42-9332d7116256">

Reply:
It's also good option but there is list of dependencies paths and each has much more chars on dependency name which isn't readability

Reply:
ok
go ahead with your approach, but
- move magic 3 to const
- use `<=` instead
- add documentation comment why do you have it the method

Reply:
The result is:
a -> ... -> c

d -> ... -> f

Reply:
@artem-fedorov fixed